package.json

```diff
@@ -27,21 +29,26 @@
     "author": "SAP SE",
     "license": "Apache-2.0",
     "dependencies": {
-        "@openui5/sap.ui.fl": "^1.81.0",
-        "@sap/cf-tools": "^0.6.8",
-        "@ui5/fs": "^2.0.3",
-        "@ui5/logger": "^2.0.0",
-        "adm-zip": "^0.4.14",
+        "@buxlabs/amd-to-es6": "^0.16.0",
+        "@rollup/plugin-node-resolve": "^11.1.0",
+        "@sap/cf-tools": "^0.8.0",
+        "@ui5/cli": "^2.8.0",
```

Comment:
do you really need the @ui5/cli as dependency or should this rather be - you should be careful what you really need as dev dependency (for the local build) and what you need as regular dependency for the runtime of your code. E.g. also the rollup parts are IMO just devDependencies, right?

Reply:
Right, moved rollup dependencies to dev.